dom.bs

```diff
@@ -5089,6 +5095,23 @@ method, when invoked, must run these steps:
  <li><p><var>document</var>'s <a>origin</a> is the <a>origin</a> of the <a>context object</a>'s
  associated <a>document</a>. [[!HTML]]
 
+ <li><p><var>document</var>'s <a for=Document>content type</a> is determined by
+ <var>namespace</var>:
+
+ <dl class=switch>
+  <dt><a>HTML namespace</a>
+  <dd><code>application/xhtml+xml</code>
+
+  <dt><a>SVG namespace</a>
+  <dd><code>image/svg+xml</code>
+
+  <dt><a>MathML namespace</a>
+  <dd><code>application/mathml+xml</code>
```

Comment:
Per https://github.com/whatwg/dom/issues/217#issuecomment-209438878 it seems this should be application/xml